code/modules/mob/living/carbon/slime/combat.dm

```diff
@@ -48,6 +48,8 @@
 		to_chat(src, "<span class='notice'>You can't attack while eating.</span>")
 		return
 
+	log_attack("<font color='red'>([src.key])[src] attacked ([target.key])[target]!</font>")
```

Comment:
Well diddle my twiddle, I just looked for other attacklog examples, It should be fine now, thanks for the knowledge.